src/components/z-card/index.tsx

```diff
@@ -29,43 +29,19 @@ export class ZCard {
   @Prop()
   coverIcon: string;
 
-  /** Enable click interactions on the card. Default: false */
-  @Prop({reflect: true})
-  clickable = false;
-
   /** Enable shadow. Default: false. */
   @Prop({reflect: true})
   showshadow = false;
 
+  /** Enable style when hover. Default: false. */
+  @Prop({reflect: true})
+  showhover = false;
```

Comment:
lo so che `showshadow` è scritto tutto in lowercase ma non è un buon esempio da cui prendere spunto. come regola per le prop abbiamo il camelCase quindi questa dovrebbe essere `showHover`. già che spacchiamo tutto forse cambierei anche `showshadow` in `showShadow`

Reply:
l'avevo cambiato 40 volte disp. okay ha senso sistemo